test/integration/framework/controlplane_utils.go

```diff
@@ -176,7 +186,7 @@ func startAPIServerOrDie(controlPlaneConfig *controlplane.Config, incomingServer
 
```

Comment:
to make it easier for tests that need an unprivileged user, define an `UnprivilegedToken` package constant in this file (with a string `unprivileged-user` or something similar), and add an additional user to the `tokens` map above:

```diff
	tokens[privilegedLoopbackToken] = &user.DefaultInfo{
		Name:   user.APIServerUser,
		UID:    uuid.New().String(),
-		Groups: []string{user.SystemPrivilegedGroup},
+		Groups: []string{user.SystemPrivilegedGroup, user.AllAuthenticated},
	}
+	tokens[UnprivilegedToken] = &user.DefaultInfo{
+		Name:   "unprivileged",
+		UID:    uuid.New().String(),
+		Groups: []string{user.AllAuthenticated},
+	}
```

Reply:
fixed